config/maintainers.json

```diff
@@ -13,7 +13,7 @@
     },
     {
       "github_username": "mcmillhj",
-      "alumnus": false,
+      "alumnus": "alumnus",
```

Comment:
Hey @mcmillhj I think the value should be `true` instead of `alumnus`, do you want me to fix it?

Reply:
@snahor I thought the same thing at first, but from the docs it looks like the only accepted values are `alumnum`, `alumnus` or `alumna`: https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md

Reply:
Hmm, I guess that document is outdated. I've checked other repos and they are using true/false.

Reply:
hmm okay, I'll update to a boolean one second.